include/rdma/fabric.h

```diff
@@ -121,6 +121,7 @@ typedef struct fid *fid_t;
 #define FI_TRANSMIT_COMPLETE	(1ULL << 27)
 #define FI_DELIVERY_COMPLETE	(1ULL << 28)
 
+#define FI_SELECTIVE_COMPLETION	(1ULL << 55)
```

Comment:
See this comment near the top of fabric.h:

/*
 * Flags
 * The 64-bit flag field is used as follows:
 * 1-grow up    common (usable with multiple operations)
 * 59-go down   operation spceific (used for single call/class)
 * 60 - 63      provider specific
 */

complete with a misspelling of specific.  If you're still in the code, can you fix the spelling at part of the same commit?